flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/combined/NtTableBuilder.java

```diff
@@ -0,0 +1,50 @@
+package com.flash3388.frc.nt.combined;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.NetworkTableValue;
+
+public class NtTableBuilder {
```

Comment:
Flashlib convention for builders is to make them as inner classes for the class they build.

Reply:
I kinda like that, but only from the user perspective